### ISSUE TITLE
Add Double Click to copy codes when they are hidden

### ISSUE THREAD

**Issue by CEOXeon** [jamie-mh/AuthenticatorPro#838]
**Is your feature request related to a problem? Please describe.**
I am always frustrated when my clipboard is overfull with OTP Codes when I just want to have a look on the hidden codes.

**Describe the solution you'd like**
Maybe you can add that you have to double click the codes to copy them when they are hidden. Because when I'm sitting at my PC and need an OTP Code I don't want to copy them to the clipboard. But on the other side when I use my phone I want to copy so that I haven't to remember the code. Therefore I'm lazy and don't want to disable and enable the setting to copy the code all the time just to get no overfull clipboard I would really like when you can double click the code when it is hidden to copy.

**TLDR**
When Code is Hidden:
* 1 Click: Reveal the Code
* 2 Clicks: Reveal and Copy the Code

When Code is not Hidden(Default):
* 1 Click: Copy the Code

**Comment by jamie-mh** [comment]
Added in 1.24.0.

**Comment by eltos** [comment]
This might be a nice feature for some people, but for others (like me) it is not.

@jamie-mh Please make this feature optional and don't force everybody to double tap!

Reason: I just failed to login multiple times, thinking I hadn't been fast enough switching between apps, until I realised codes are no longer copied when tapped and I had been using an old code all the time. Took a while until I read the note in the change log and realised I now have to double tap every time.
This is actually quite annoying, because I almost always just copy-paste the code without even reading it. I don't mind the cases where I manually type it and the code is left in the clipboard unused, since my clipboard auto-clears itself anyways.  
TL;DR: The App is great, please allow us to keep the old behaviour, and make the double-tab feature optional for people that prefer it.

**Comment by MmyGgithubAaccount** [comment]
I'd also like to see single tap to copy implemented
